.circleci/defaults.sh

```diff
@@ -44,3 +47,5 @@ mapping["boom"]="SUB_PROJECT=example CONFIG=SmallBoomConfig"
 mapping["rocketchip"]="SUB_PROJECT=rocketchip"
 mapping["blockdevrocketchip"]="SUB_PROJECT=example CONFIG=SimBlockDeviceRocketConfig TOP=TopWithBlockDevice"
 mapping["hwacha"]="SUB_PROJECT=example CONFIG=HwachaRocketConfig GENERATOR_PACKAGE=hwacha"
+mapping["firesim"]="DESIGN=FireSim TARGET_CONFIG=FireSimRocketChipConfig PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig"
+mapping["fireboom"]="DESIGN=FireBoom TARGET_CONFIG=FireSimSmallBoomConfig PLATFORM_CONFIG=FireSimDDR3FRFCFSLLC4MBConfig"
```

Comment:
Why not the large one?

Reply:
Takes too long

Reply:
i'd rather use a config that we're actually building into AGFIs. Can we run a smaller subset of tests or something?